src/net_permissions.h

```diff
@@ -84,6 +84,10 @@ class NetWhitebindPermissions : public NetPermissions
 public:
     static bool TryParse(const std::string& str, NetWhitebindPermissions& output, bilingual_str& error);
     CService m_service;
+
+    bool operator<(const NetWhitebindPermissions& other) const {
+        return m_service < other.m_service;
```

Comment:
isn't it confusing that in case of service duplicates the it's not obvious which `m_flags` is ignored?

What do we expect to happen in case of something like `-whitebind=127.0.0.1:8335=relay,noban -whitebind=127.0.0.1:8335=relay` vs `-whitebind=127.0.0.1:8335=relay -whitebind=127.0.0.1:8335=relay,noban`?
Or even worse, `-whitebind=127.0.0.1:8335=all -whitebind=127.0.0.1:8335=relay,mempool`?